archinstall/lib/installer.py

```diff
@@ -363,8 +364,10 @@ def add_bootloader(self, bootloader='systemd-bootctl'):
 				boot_partition = partition
 			elif partition.mountpoint == self.target:
 				root_partition = partition
-
-		self.log(f'Adding bootloader {bootloader} to {boot_partition}', level=logging.INFO)
+		if hasUEFI():
```

Comment:
I'm confused why we are even checking hasUEFI here if we do the same self.log regardless...?

Reply:
Ohhhhhh it's root partition or boot partition. Nevermind this comment.